modules/bmo-editing-the-hostfirmwaresettings-resource.adoc

```diff
@@ -33,7 +38,49 @@ Where `<host_name>` is the name of a provisioned host. The `HostFirmwareSettings
 ----
 spec:
   settings:
-    name: value
+    name: value <1>
 ----
+<1> Use the `FirmwareSchema` resource to identify the available settings for the host. You cannot set values that are read-only.
 
 . Save the changes and exit the editor.
+
+. Annotate the host to delete it from the machineset:
++
+[source,terminal]
+----
+$ oc annotate machine <host_name> machine.openshift.io/cluster-api-delete-machine=yes -n openshift-machine-api
+----
++
+Where `<host_name>` is the name of the host to delete.
+
+. Get a list of nodes and count the number worker nodes:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+. Get the machineset:
++
+[source,terminal]
+----
+$ oc get machinesets -n openshift-machine-api
+----
+
+. Scale the machineset:
++
+[source,terminal]
+----
+$ oc scale machineset <machineset_name> -n openshift-machine-api --replicas=<n>
+----
++
+Where `<machineset_name>` is the name of the machineset and `<n>` is the decremented number of worker nodes.
+
+. When the host enters the `Available` state, scale up the machineset to make the `HostFirmwareSettings` resource changes take effect:
++
+[source,terminal]
+----
+$ oc scale machineset <machineset_name> -n openshift-machine-api --replicas=<n>
+----
++
+Where `<machineset_name>` is the name of the machineset and `<n>` is the incremented number of worker nodes.
```

Comment:
I would use n-1 in the first scaling and n here with explanation `<n>` is the number of worker nodes